userguide/sound_classifier/how-it-works.md

```diff
@@ -0,0 +1,52 @@
+# How Does this Work?
+
+Training and making predctions, for a sound classifier model, is a three
+stage process. The first stage does signal preprocessing. In
+the second step we use a pretrained neural network to extract deep
+features. Finally in the last step, we use a custom neural network.
+
+## Signal Preprocessing Pipeline Stage
+Several audio transformation happen during this stage. For someone without
+sound domain knowledge, this is probably the most complicated part.
+Nothing about this stage is updated based on the input data (i.e. nothing
+is learned in this stage).
+
+At a high level the preprocessing pipeline does the following:
```

Comment:
Insert comma between "high level" and "the"?

Reply:
Done.